src/containers/PipelineResourcesModal/PipelineResourcesModalErrorNotification.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { fireEvent, waitForElement } from 'react-testing-library';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { renderWithIntl } from '../../utils/test';
+import PipelineResourcesModal from '.';
+import * as API from '../../api';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+const namespaces = {
+    byName: {
+      default: {
+        metadata: {
+          name: 'default',
+          selfLink: '/api/v1/namespaces/default',
+          uid: '32b35d3b-6ce1-11e9-af21-025000000001',
+          resourceVersion: '4',
+          creationTimestamp: '2019-05-02T13:50:08Z'
+        }
+      }
+    },
+    errorMessage: null,
+    isFetching: false,
+    selected: 'default'
+  };
+
+const byNamespace = {
+    default: [
+      {
+        uid: '0',
+        name: 'github-repo-access-secret',
```

Comment:
Isn't this actually a PipelineResource name?

Reply:
ahh yep - fixed now 😸